ss/ss.go

```diff
@@ -18,24 +19,14 @@ import (
 )
 
 const (
-	processeNameFieldIdx  = 5
 	localAddrPortFieldIdx = 3
 	interval              = time.Millisecond * 500
 	duration              = time.Second * 5
 )
 
-var (
-	// TcpSSFilterFn is a function variable in Go that filters entries from the 'ss' command output.
-	// It takes an entry from the 'ss' command output and returns true if the entry represents a TCP port in the listening state.
-	tcpSSFilterFn = func(s string) bool {
-		return strings.Contains(s, "127.0.0") || !strings.Contains(s, "LISTEN")
-	}
-	// UdpSSFilterFn is a function variable in Go that filters entries from the 'ss' command output.
-	// It takes an entry from the 'ss' command output and returns true if the entry represents a UDP port in the listening state.
-	udpSSFilterFn = func(s string) bool {
-		return strings.Contains(s, "127.0.0") || !strings.Contains(s, "ESTAB")
-	}
-)
+var filterOutFn = func(s string) bool {
+	return strings.Contains(s, "127.0.0") || strings.Contains(s, "::1") || s == "" || strings.Contains(s, "State")
```

Comment:
What is `State` is that from the headers? Please note I updated the command to avoid headers https://github.com/openshift-kni/commatrix/pull/5/files#diff-84c58a8a915a8e7674df707f44568d303d60ffd74e96e56637084e566d72562cR50

Reply:
done, added the "H" flag here as well.